pkg/client/dtclient/config_client.go

```diff
@@ -275,20 +276,21 @@ func stripCreateOnlyPropertiesFromAppMobile(payload []byte) []byte {
 // retrying in case of know errors on upload.
 func (d *DynatraceClient) callWithRetryOnKnowTimingIssue(ctx context.Context, restCall rest.SendRequestWithBody, path string, body []byte, theApi api.API) (rest.Response, error) {
 	resp, err := restCall(ctx, path, body)
-
 	if err == nil && resp.IsSuccess() {
 		return resp, nil
 	}
-
 	if err != nil {
 		log.WithCtxFields(ctx).WithFields(field.Error(err)).Warn("Failed to send HTTP request: %v", err)
 	} else {
 
 		log.WithCtxFields(ctx).WithFields(field.F("statusCode", resp.StatusCode)).Warn("Failed to send HTTP request: (HTTP %d)!\n    Response was: %s", resp.StatusCode, string(resp.Body))
 	}
 
-	var setting rest.RetrySetting
+	if isCalculatedLogMonitoringAPIDisabled(resp) {
```

Comment:
since `callWithRetryOnKnownTimingIssues` is now also called from withing `getExistingValuesFromEndpoint` we run into that error for which it does not make sense to do any retry.